student/src/main/java/nl/quintor/studybits/student/Seeder.java

```diff
@@ -1,73 +0,0 @@
-package nl.quintor.studybits.student;
```

Comment:
I added this class to quickly spin up and bootstrap the frontend. If it interferes with something you want to be doing, can you make the bean conditional on a profile (called "demo") instead of deleting the entire class?

Reply:
Oh I see! I thought that I added it way back when. I'll add it again, no problem.